server/test_test.go

```diff
@@ -200,6 +201,7 @@ func (c *cluster) shutdown() {
 			os.Remove(cf)
 		}
 		if sd != _EMPTY_ {
+			sd = strings.TrimSuffix(sd, JetStreamStoreDir)
```

Comment:
@derekcollison I noticed that the temporary directory `nats-server` would still be filled with empty directory, such as `jetstream12345678`. Since it seems that under this directory, the server would actually create `jetstream` directory, this remove all would remove `jetstream` but leave the empty parent directory.
This change helps cleaning up, and seems that JetStream pass, but keep that in mind if later when writing a test in case this change has an effect (for instance if you were expecting only `jetstream` to be removed but not the parent dir, etc..).